src/systemcmds/motor_test/motor_test.cpp

```diff
@@ -88,7 +88,7 @@ Note: this can only be used for drivers which support the motor_test uorb topic
 
 	PRINT_MODULE_USAGE_NAME("motor_test", "command");
 	PRINT_MODULE_USAGE_COMMAND_DESCR("test", "Set motor(s) to a specific output value");
-	PRINT_MODULE_USAGE_PARAM_INT('m', -1, 0, 7, "Motor to test (0...7, all if not specified)", true);
+	PRINT_MODULE_USAGE_PARAM_INT('m', -1, 1, 8, "Motor to test (1...8, all if not specified)", true);
```

Comment:
Can we keep 0 indexing here?

Reply:
@jlecoeur I think having this command with 1..8 range is more compliant with what is specified in the documentation/reference images ( Motor 1, Motor 2 and so on, https://docs.px4.io/master/en/airframes/airframe_reference.html#quadrotor-x). In this view things are more uniform, no?